tasklist/els-schema/src/main/java/io/camunda/tasklist/store/opensearch/FormStoreOpenSearch.java

```diff
@@ -101,12 +102,16 @@ public Optional<FormIdView> getFormByKey(final String formKey) {
       final GetResponse<FormIdView> response = osClient.get(request, FormIdView.class);
       if (response.found() && response.source() != null) {
         return Optional.of(response.source());
-      } else {
+      }
+    } catch (final OpenSearchException e) {
+      if (e.response().status() == HttpURLConnection.HTTP_NOT_FOUND) {
```

Comment:
I would add a debug log for the exception, you never know (maybe the 404 is returned because the index does not exist)